packages/sdk/identity/src/odis/identifier.ts

```diff
@@ -0,0 +1,217 @@
+import {
+  CombinerEndpointPNP,
+  KEY_VERSION_HEADER,
+  SignMessageRequest,
+  SignMessageResponseSchema,
+} from '@celo/phone-number-privacy-common'
+import { soliditySha3 } from '@celo/utils/lib/solidity'
+import { createHash } from 'crypto'
+import debugFactory from 'debug'
+import { BlsBlindingClient, WasmBlsBlindingClient } from './bls-blinding-client'
+import {
+  AuthenticationMethod,
+  AuthSigner,
+  EncryptionKeySigner,
+  getOdisPnpRequestAuth,
+  queryOdis,
+  ServiceContext,
+} from './query'
+
+const debug = debugFactory('kit:odis:identifier')
+const sha3 = (v: string) => soliditySha3({ type: 'string', value: v })
+
+const PEPPER_CHAR_LENGTH = 13
+const PEPPER_SEPARATOR = '__'
+
+// using DID methods as prefixes when they exist
+// https://w3c.github.io/did-spec-registries/#did-methods
+export enum IdentifierPrefix {
+  PHONE_NUMBER = 'tel',
+  EMAIL = 'mailto',
+  TWITTER = 'twit',
+  // feel free to put up a PR to add more types!
+}
+
+// plaintext identifier: off-chain information, ex: phone number, twitter handle, email, etc.
+// blinded identifier: obtained by blinding the plaintext identifier
+// blinded signature: blinded identifier signed by ODIS
+// unblinded signatue: obtained by unblinding the blinded signature
+// pepper: unique secret, obtained by hashing the unblinded signature
+// obfuscated identifier: identifier used for on-chain attestations, obtained by hashing the plaintext identifier and pepper
+
```

Comment:
Putting this into a [TSDoc](https://tsdoc.org/) comment will allow it to appear in the online docs.
```suggestion
/** Details with the private plaintext identifier and obfuscated identifier, which can be made public.
 * plaintext identifier: off-chain information, ex: phone number, twitter handle, email, etc.
 * blinded identifier: obtained by blinding the plaintext identifier
 * blinded signature: blinded identifier signed by ODIS
 * unblinded signatue: obtained by unblinding the blinded signature
 * pepper: unique secret, obtained by hashing the unblinded signature
 * obfuscated identifier: identifier used for on-chain attestations, obtained by hashing the plaintext identifier and pepper
 */
```

Reply:
good suggestion thanks! I've committed this change but I think they are actually planning on completely taking down the sdk docs site